gapic/samplegen/samplegen.py

```diff
@@ -1062,7 +1055,7 @@ def generate_sample_specs(api_schema: api.API, *, opts) -> Generator[Dict[str, A
                 yield spec
 
 
-def generate_sample(sample, api_schema, sample_template: jinja2.Template) -> str:
+def generate_sample(sample, api_schema, sample_template: jinja2.Template) -> Tuple[str, Any]:
```

Comment:
Why use `Any` as the type of the second second element instead of the real type? Is it related to the `# type: ignore` a few lines down?

Reply:
Yes, since the type is ignored I cannot use it in an annotation. I learned about [mypy](https://github.com/dropbox/mypy-protobuf) from a PR with Tres though, so I'll open a separate PR to start type checking all the protobuf types.